.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 # Primary repo maintainers
-*       @noot @arijitAD @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya @omar391
\ No newline at end of file
+*       @noot @arijitAD @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya
```

Whilst we're at it :wink: 

```suggestion
*       @noot @arijitAD @edwardmack @timwu20 @EclesioMeloJunior @jimjbrettj @kishansagathiya @qdm12
```

---

@timwu20 yes pls do this!!